api/Dockerfile

```diff
@@ -1,5 +1,8 @@
 FROM node:16-alpine
-RUN apk add --no-cache fontconfig
+
+RUN apk add --no-cache --virtual .gyp python3 make g++
+RUN apk --no-cache --update add fontconfig ttf-dejavu
```

Comment:
just curious, why is `--virtual` and `--update` needed at these lines?

Reply:
Sorry but I didn't look enough into this. I sampled these from a few different sources for people who had problems with `pango` (a dependency for `fontconfig`) and missing fonts. I imagine we don't need either here but I saw a few posts similar to this. Hope it's just some best practices that I don't fully understand.